### DISCUSSION TITLE
Unable to create organization

### DISCUSSION THREAD

**Discussion by Bot125388** [community/community#121250] [Enterprise]
I created an enterprise in vSphere and configured it but once I attempted to create an organization, the button was disabled and I am unable to create an organization. I have found no one having the same problem on the internet.

**Comment by queenofcorgis** [comment]
Hi there @Bot125388 👋🏼 

Are you trying to create a GitHub organization? You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by kww4** [comment]
> Hi there @Bot125388 👋🏼 
> 
> 
> 
> Are you trying to create a GitHub organization? You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.
